src/deluge/gui/l10n/l10n.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
```

Comment:
as we don't use `.h` vs `.hpp` for the most part for C vs C++ headers, perhaps could be called out more explicitly in the name as well, like `l10n_for_c.h`. And/or explained in a short comment. I was confused at first "why just four strings??".

Reply:
Yeah, this is only really used by the RZA1 library in a couple places anyways. Maybe I should just move it over there, since that's all C anyways? Something like RZA1/extern/deluge_l10n.h

Common C headers are something I'd usually put in {project_root}/include, especially if they needed to be accessed by other internal-project libraries.

Reply:
or heck they should go in deluge.h probably

Reply:
Fixed